support-frontend/app/admin/settings/Switches.scala

```diff
@@ -22,23 +25,39 @@ case class FeatureSwitches(
     authenticateWithOkta: Option[SwitchState],
 )
 
+object FeatureSwitches {
+  implicit val featureSwitchesCodec = deriveCodec[FeatureSwitches]
```

Comment:
missing type annotaion
```suggestion
  implicit val featureSwitchesCodec: Codec[FeatureSwitches] = deriveCodec
```